src/gui/widgets/size_lock.hpp

```diff
@@ -0,0 +1,136 @@
+/*
+   Copyright (C) 2016 Jyrki Vesterinen <sandgtx@gmail.com>
+   Part of the Battle for Wesnoth Project http://www.wesnoth.org/
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY.
+
+   See the COPYING file for more details.
+*/
+
+#ifndef GUI_WIDGETS_SIZE_FIXATER_HPP_INCLUDED
+#define GUI_WIDGETS_SIZE_FIXATER_HPP_INCLUDED
+
+#include <gui/widgets/container.hpp>
+
+#include <gui/auxiliary/formula.hpp>
+#include <gui/core/widget_definition.hpp>
+#include <gui/core/window_builder.hpp>
+#include <gui/widgets/generator.hpp>
+
+namespace gui2
+{
+
+	namespace implementation
```

Comment:
Non-standard indentation. Please drop the indentation level inside namespace gui2.